src/Common/src/Interop/Unix/libcurl/Interop.libcurl_multi.cs

```diff
@@ -52,5 +53,8 @@ internal static partial class libcurl
         [DllImport(Interop.Libraries.LibCurl)]
         public static extern IntPtr curl_multi_strerror(
             int code);
+
+        [DllImport(Libraries.CryptoNative)]
```

Comment:
I initially tried to put it in Interop.Crypto.cs , but that is used in many other projects (eg: System.Cryptography.Primitive) who have no interest in ```SafeSslHandle``` and ```SafeX509Handle```. Bringing the needed dependencies into Cryptography.Primitive is too cumbersome for future changes/refactoring as well.

But if you are suggesting that I introduce a new file for this function, I can do it. Just that no one else uses it, and I wanted to guard it against refactoring :).